llvm/test/CodeGen/RISCV/GlobalISel/alu-roundtrip.ll

```diff
@@ -1,8 +1,8 @@
 ; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
 ; RUN: llc -mtriple=riscv32 -mattr=+m -global-isel -verify-machineinstrs < %s \
-; RUN:   | FileCheck %s --check-prefix=RV32IM
+; RUN:   -riscv-enable-copy-propagation=false | FileCheck %s --check-prefix=RV32IM
```

Comment:
I pulled down your PR, built llc, changed this option to true, and there was no impact on the diff. I am confused what you mean about see through the ADDI when we have a sub instr.